code/web/src/ui/image/Tile.js

```diff
@@ -16,6 +16,7 @@ const Tile = (props) => {
           background-image:url('${ image }');
           background-size: 100% auto;
           box-shadow: ${ shadow ? shadow : 'none' };
+          opacity: ${opacity ? opacity : 1}
         }
```

Comment:
Great job on this. I think this is a nice ui feature and I'm glad we were able to get it implemented.